### ISSUE TITLE
Installation Problem - Please install application dependencies via composer (or manually, if you dislike it ;)

### ISSUE THREAD

**Issue by xristinal** [tkrebs/ep3-bs#610]
Hello,
Ι am trying to Install ep3-bs-1.7 and the bellow problem:
**Please install application dependencies via composer (or manually, if you dislike it ;)**
Version php 7.4
I follow exatcly the instructions from website. https://bs.hbsys.de/documentation/en/installation/setup
I tried 3 times from the beginning the installation. The same result and indeed on a different server.

I have make installation this ep3-bs application many times at the past. Ι decided to upgrade my systems for server compatibility reasons. I don't know which reason happens that.
I would appreciate your help, what steps to follow to fix this. It is important to my community.

Best Regards,
Xristi

**Comment by xristinal** [comment]
Yes.. I reply the steps that I followed:
**1st attempt:**
I download the **download zip** from github. 
<img width="500" alt="Screen Shot 2023-09-01 at 10 48 59 PM" src="https://github.com/tkrebs/ep3-bs/assets/37621113/ead9927a-d5f6-469f-9c6f-b9cd33b34470">
I upload it this zip file in my web server, I follow the steps from the Installation instructions.

**2nd attempt:**
I was tried via [Git](http://git-scm.com/). i installed it in my laptop locally, I got a clone file from https://github.com/tkrebs/ep3-bs and then I update in my web server. The same problem.

**3nd attempt:**
I installed Composer program in my laptop locally. I was trying  to install the ep3-bs system. no success

I would like to give us some extra info or step to install this system. I am trying to install it in the web server, no locally.
In the past, the installation was so easy.. what has changed since the new versions? 

I would appreciate it if you could provide us with additional information.
Thank you in advance.

**Comment by cbits68** [comment]
And did you ever used composer to install dependencies?
What about the folder vendor? Is it empty?
If you wanted to install via git way you have to use composer to get the required modules for Zend Framework.

There is also a zip file available including all required modules you can use without loading them by your own with composer. You can try this one, that already includes all neccesary data in the vendor folder.

**Comment by olemde** [comment]
Hi xristinal. did you solved the problem . I am in front of  the same situation. But opposite to you i have any idea what this "composer" is or means...

br
Ole

**Comment by xristinal** [comment]
Yes! I found a solution. You have to download the installation file from website https://bs.hbsys.de/download
![installation](https://github.com/tkrebs/ep3-bs/assets/37621113/5f69454b-e1c7-42ce-8668-f6037c4c20e7)
and then follow the installation instructions from website.https://bs.hbsys.de/documentation/en/installation/setup
